src/main/java/data/SheetsConverter.java

```diff
@@ -61,59 +62,70 @@ public void writeToSheet(ArrayList<Vendor> vendorList, Sheets sheetsService)
       .values().update(SPREADSHEET_ID, VENDOR_RANGE, vendorBody)
       .setValueInputOption(INPUT_OPTION).execute();
   }
-
+ 
   /** Clear all data from the Account and Vendor spreadsheets. */
   public void clearSheets(Sheets sheetsService) throws IOException {
     ClearValuesRequest requestBody = new ClearValuesRequest();
-
+ 
     ClearValuesResponse clearAccount =  sheetsService.spreadsheets().values()
       .clear(SPREADSHEET_ID, TAB_1, requestBody).execute();
     ClearValuesResponse clearVendor = sheetsService.spreadsheets().values()
       .clear(SPREADSHEET_ID, TAB_2, requestBody).execute();
   } 
-
+ 
   /** Store the Accounts(value) associated to its Vendor(key) in a HashMap. */
   public HashMap<String, ArrayList<Account>> getAllAccounts(
       ArrayList<Vendor> vendors) {      
-
+ 
     HashMap<String, ArrayList<Account>> allAccounts =  
       new HashMap<String, ArrayList<Account>>();
-
+ 
     for(int i = 0; i < vendors.size(); i++) {
       allAccounts.put(vendors.get(i).getVendorID(), vendors.get(i).getAccounts());
     }
-
+ 
     return allAccounts; 
   }
-
+ 
   public List<List<Object>> buildAccountSheetBody(
-      HashMap<String, ArrayList<Account>> accounts) {
+      HashMap<String, ArrayList<Account>> vendorIdMap) {
 
-    // TODO: @cfloeder Sort the accounts by some criteria x. 
     List<List<Object>> accountSheetData = new ArrayList<List<Object>>();
-    accountSheetData.add(new ArrayList<Object>(Account.getAccountSheetHeader()));
-
-    for(Map.Entry<String, ArrayList<Account>> entry : accounts.entrySet()) { 
-      ArrayList<Account> accountList = entry.getValue();
-      for(int i = 0; i < accountList.size(); i++) {
-        accountSheetData.add(new ArrayList<Object>(accountList.get(i)
-        .getAccountSheetsRow(entry.getKey())));
+ 
+    // Sort the Account Sheet aphabetically by VendorID.
+    ArrayList<String> sortedKeys = sortKeys(vendorIdMap);
+ 
+    for(String vendorID : sortedKeys) {
+      ArrayList<Account> accountList = vendorIdMap.get(vendorID);
+      for(Account account : accountList) {
+        accountSheetData.add(new ArrayList<Object>(account
+          .getAccountSheetsRow(vendorID)));
       }
     }
 
+    accountSheetData.add(0, new ArrayList<Object>(Account.getAccountSheetHeader()));
+ 
     return accountSheetData;
   }
-
+ 
+  public ArrayList<String> sortKeys(HashMap<String, ArrayList<Account>> vendorIdMap) {
```

Comment:
I would just inline this as Collections.sort(vendorIdMap.keySet()) and remove this method

Reply:
Sounds good